### PR TITLE
Part 4 - XML parsing

### PR DESCRIPTION
**Pull request by benjimarshall**

### CODE REVIEW

File: SupportBank/CliInterface.cs [hunk 1]
```diff
@@ -96,6 +96,10 @@ private static void ReadFromFile(string filename, AllPeople people)
                     Program.Logger.Debug($"Working on JSON: {filename}");
                     JsonReader.ReadJson(filename, people);
                     break;
+                case "xml":
+                    Program.Logger.Debug($"Working on : {filename}");
```
Comment (FridaTveit) [inline SupportBank/CliInterface.cs:100]: Do you want "XML" in this log, to be consistent with the CSV and JSON one? 🙂

**Review by FridaTveit — COMMENTED** [review]
One tiny thing, otherwise looks good 🙂

File: SupportBank/CliInterface.cs (hunk 1 above)
Comment (benjimarshall) [reply SupportBank/CliInterface.cs]: Yep, I did mean to change that but mustn't have managed it! Fixed in 9049c77 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**